app/dashboard/models.py

```diff
@@ -601,23 +613,34 @@ def value_in_eth(self):
         except Exception:
             return None
 
-    # TODO: DRY
     @property
-    def value_in_usdt(self):
+    def value_in_usdt_now(self):
         decimals = 1
         if self.tokenName == 'USDT':
             return float(self.amount)
         if self.tokenName == 'DAI':
             return float(self.amount / 10**18)
         try:
-            return round(float(convert_amount(self.value_in_eth, 'ETH', 'USDT')) / decimals, 2)
-        except Exception:
+            return round(float(convert_amount(self.amount, self.tokenName, 'USDT')) / decimals, 2)
+        except ConversionRateNotFoundError:
+            return None
+
+    @property
+    def value_in_usdt(self):
+        decimals = 1
+        if self.tokenName == 'USDT':
+            return float(self.amount)
+        if self.tokenName == 'DAI':
+            return float(self.amount / 10 ** 18)
+        try:
+            return round(float(convert_amount(self.amount, self.tokenName, 'USDT', self.received_on)) / decimals, 2)
```

Comment:
`self.received_on`

can u pls use `self.created_on` intead?  not all tips have been received